parameter_set/CMakeLists.txt

```diff
@@ -0,0 +1,90 @@
+cmake_minimum_required(VERSION 3.5)
+project(parameter_set)
+
+# Default to C++17
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
+endif()
+
+# Compile options set for the targets
+set(PRIVATE_COMPILE_OPTIONS -Wall -Wextra -Wpedantic)
+
+# find dependencies
+find_package(absl_status REQUIRED)
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_ros REQUIRED)
+find_package(rcl REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(sensor_msgs REQUIRED)
+
+set(THIS_PACKAGE_INCLUDE_DEPENDS
+  rclcpp
+  sensor_msgs
+  absl_status
+)
+
+add_library(${PROJECT_NAME}
+  src/parameter_set.cpp
+  src/parameter_descriptor_builder.cpp
+  src/validate_parameter.cpp
+)
+target_include_directories(${PROJECT_NAME} PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include>)
+target_compile_options(${PROJECT_NAME} PRIVATE ${PRIVATE_COMPILE_OPTIONS})
+ament_target_dependencies(${PROJECT_NAME} ${THIS_PACKAGE_INCLUDE_DEPENDS})
+
+add_executable(example_node
+  src/example_node/main.cpp
+  src/example_node/robot_parameters.cpp
+  src/example_node/robot_subsystem.cpp
+)
+ament_target_dependencies(example_node ${THIS_PACKAGE_INCLUDE_DEPENDS})
+target_link_libraries(example_node ${PROJECT_NAME})
+
+# Install Libraries
+install(
+  TARGETS ${PROJECT_NAME}
+  EXPORT ${PROJECT_NAME}Targets
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION bin
```

Comment:
Fix the runtime path to `lib/${PROJECT_NAME}` and add `example_node` here

Reply:
done